tests/integration/components/uni-datepicker-test.js

```diff
@@ -6,9 +6,45 @@ moduleForComponent('uni-datepicker', 'Integration | Component | uni datepicker',
 });
 
 test('it renders', function(assert) {
-  assert.expect(1);
+  this.set('messages', [
+    { type: 'information', label: 'This is a information label' },
```

Comment:
Use the enum types here, possibly even moving the messages to consts and reuse them across all tests.
Applies to all tests below.